GW2EIEvtcParser/EncounterLogic/Raids/W1/ValeGuardian.cs

```diff
@@ -151,36 +151,31 @@ internal override void ComputeEnvironmentCombatReplayDecorations(ParsedEvtcLog l
 
         if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.ValeGuardianDistributedMagic, out var distributedMagicEvents))
         {
-            int distributedMagicDuration = 6700;
-            //knownEffectsIDs.Add(distributedMagicGUIDEvent.ContentID);
             foreach (EffectEvent distributedMagic in distributedMagicEvents)
             {
-                int start = (int)distributedMagic.Time;
-                int expectedEnd = start + distributedMagicDuration;
-                int end = Math.Min(expectedEnd, (int)distributedMagic.Src.LastAware);
-                var circle = new CircleDecoration(180, (start, end), Colors.Green, 0.2, new PositionConnector(distributedMagic.Position));
-                EnvironmentDecorations.Add(circle);
-                EnvironmentDecorations.Add(circle.Copy().UsingGrowingEnd(expectedEnd));
+                // Effect duration is 6000, added 700 for the damage.
+                (long start, long end) lifespan = distributedMagic.ComputeLifespan(log, 6700);
+                lifespan.end = Math.Min(lifespan.end, distributedMagic.Src.LastAware);
```

Comment:
should be Min(lifespan.end + 700, LastAware), ComputeLifespan will not allow it go to above effect.Duration.
ComputeLifespan should be called with 6000.